### PR TITLE
[KHRGA-25] Updated CI Common by-value semantics

### PR DESCRIPTION
**Pull request by PetroBondar**
Issue: KHRGA-25

### CODE REVIEW

**Review by keryell — APPROVED** [review]
That looks good.

File: source/iface/common-byval.rst [hunk 1]
```diff
@@ -4,24 +4,135 @@
...
 
-Types: :ref:`id`, :ref:`range`, :ref:`item`, :ref:`nd_item`,
-:ref:`h_item`, :ref:`group`, `sub_group`, and :ref:`nd_range`.
+.. Add sup_group ref after creating Data Access chapter structure
 
-Classes with value semantics support the following methods.
+Each of the following SYCL runtime classes:
+:ref:`id`, :ref:`range`, :ref:`item`, :ref:`nd_item`,
+:ref:`h_item`, :ref:`group`, `sub_group`, and :ref:`nd_range`
+must follow the following statements, where ``T`` is the
```
Comment (mkinsner) [inline source/iface/common-byval.rst:16]:
```suggestion
obey follow the following statements, where ``T`` is the
```

File: source/iface/common-byval.rst [hunk 2]
```diff
@@ -4,24 +4,135 @@
...
+runtime class type:
+
+* ``T`` must be default copy constructible and copy assignable
+  on the host application (in the case where ``T`` is available on
```
Comment (mkinsner) [inline source/iface/common-byval.rst:20]:
```suggestion
  in the host application (in the case where ``T`` is available on
```
I will make a SYCL spec PR to update this word in the actual SYCL spec.  "on" -> "in" throughout this list

File: source/iface/common-byval.rst [hunk 3]
```diff
@@ -4,24 +4,135 @@
...
+  the host) and within SYCL kernel functions.
+* ``T`` must be default destructible on the host application
```
Comment (mkinsner) [inline source/iface/common-byval.rst:22]:
```suggestion
* ``T`` must be default destructible in the host application
```

File: source/iface/common-byval.rst [hunk 4]
```diff
@@ -4,24 +4,135 @@
...
+  (in the case where ``T`` is available on the host) and
+  within SYCL kernel functions.
+* ``T`` must be default move constructible and default move
+  assignable on the host application (in the case where ``T``
```
Comment (mkinsner) [inline source/iface/common-byval.rst:26]:
```suggestion
  assignable in the host application (in the case where ``T``
```

File: source/iface/common-byval.rst [hunk 5]
```diff
@@ -4,24 +4,135 @@
...
+  is available on the host) and within SYCL kernel functions.
+* ``T`` must be equality comparable on the host application
```
Comment (mkinsner) [inline source/iface/common-byval.rst:28]:
```suggestion
* ``T`` must be equality comparable in the host application
```

File: source/iface/common-byval.rst (hunk 2 above)
Comment (mkinsner) [reply source/iface/common-byval.rst:20]: https://github.com/KhronosGroup/SYCL-Docs/pull/490

File: source/iface/common-byval.rst (hunk 5 above)
Comment (PetroBondar) [reply source/iface/common-byval.rst]: Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.

File: source/iface/common-byval.rst (hunk 4 above)
Comment (PetroBondar) [reply source/iface/common-byval.rst]: Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.

File: source/iface/common-byval.rst (hunk 3 above)
Comment (PetroBondar) [reply source/iface/common-byval.rst]: Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.

File: source/iface/common-byval.rst (hunk 2 above)
Comment (PetroBondar) [reply source/iface/common-byval.rst]: Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.

File: source/iface/common-byval.rst (hunk 1 above)
Comment (PetroBondar) [reply source/iface/common-byval.rst]: Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.

**Review by mkinsner — APPROVED** [review]

**Pull request merged**